app/components/sections/hero-section.tsx

```diff
@@ -154,6 +156,18 @@ function HeroSection({
               </H2>
             </motion.div>
           ) : null}
+          <motion.div variants={childVariants}>
+            <H4 as="p" variant="secondary" className="mt-3">
+              <Link
+                prefetch="intent"
```

Comment:
```suggestion
                reloadDocument
```

We need a document reload because this isn't a client-side transition (so we can't prefetch either)